src/build-secrets/Dockerfile-arm

```diff
@@ -0,0 +1,6 @@
+FROM balenalib/armv7hf-alpine:3.10
+RUN ["cross-build-start"]
+RUN apk update && apk add bash jq
+RUN ["cross-build-end"]
```

Comment:
@CameronDiver, @nazrhom, the build of the ARM containers was failing at the `RUN apk update` step, I gather because of mismatched architecture -- balena CI presumably uses a Docker daemon running on x86, and the executables are for ARM. Can balena CI use a native ARM builder? I assume the answer is no, but I was successful replacing the base image (e.g. using `balenalib/armv7hf-alpine:3.10`) and adding these `RUN ["cross-build-start"]` and `RUN ["cross-build-end"]` as explained in [Petros' blog/tutorial](https://www.balena.io/blog/building-arm-containers-on-any-x86-machine-even-dockerhub/).